sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -467,10 +494,148 @@ The `node` class provides the {crs}[common reference semantics].
 [source,c++]
 ----
 namespace sycl::ext::oneapi::experimental {
-  class node {};
+  class node {
+    template <int Dimensions>
+    void update_nd_range(nd_range<Dimensions> executionRange);
+  };
```

Comment:
This is the second synopsis for `class node`, which is duplicated from above.  FWIW, I think it's nicer to have the synopsis here because it's closer to the description of the member functions.

Reply:
I agree that might be a nicer change, but that would should probably be applied across the extension synopsis, splitting it into the sections for each class. In that case I'll probably remove this duplicate and do that as a separate PR to avoid clutter here.

Reply:
OK.